socorro/scripts/add_crashid_to_queue.py

```diff
@@ -0,0 +1,121 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import argparse
+import os
+
+import pika
+
+from socorro.lib.ooid import is_crash_id_valid
+
+
+EPILOG = """
+To use in a docker-based local dev environment:
+
+    python scripts/add_crashid_to_queue.py socorro.normal <CRASHID>
+
```

Comment:
It's just adding some additional bits to the beginning, but that's true of running anything through docker-compose run. I'm not sure that's prudent here, but might be a good thing for the docs.